scripts/configs/nightly.yaml

```diff
@@ -0,0 +1,96 @@
+# NOTE: all these experiments should get >45 / 50 in the NUM_SOLVED column of
+# the table printed out by `python scripts/analyze_results_directory.py`,
+# except stick_button_option_learning, which should get ~40-45 / 50.
+
+# oracle, NSRT learning, and predicate invention experiments
+---
+APPROACHES:
+  oracle:
+    NAME: "oracle"
+    FLAGS: {}
+  nsrt_learning:
+    NAME: "nsrt_learning"
+    FLAGS: {}
+  invent_allexclude:
+    NAME: "grammar_search_invention"
+    FLAGS:
+      excluded_predicates: "all"
+ENVS:
+  cover:
+    NAME: "cover"
+    FLAGS: {}
+  pybullet_blocks:
+    NAME: "pybullet_blocks"
+    FLAGS: {}
+  painting:
+    NAME: "painting"
+    FLAGS: {}
+  tools:
+    NAME: "tools"
+    FLAGS:
+      num_train_tasks: 200  # requires more data (default is 50)
+  pybullet_cover:
+    NAME: "pybullet_cover"
+    FLAGS: {}
+ARGS: []
+FLAGS: {}
+START_SEED: 456
+NUM_SEEDS: 10
+...
+
+# other environments with oracle and NSRT learning
+---
+APPROACHES:
+  oracle:
+    NAME: "oracle"
+    FLAGS: {}
+  nsrt_learning:
+    NAME: "nsrt_learning"
+    FLAGS: {}
+ENVS:
+  playroom:
+    NAME: "playroom"
+    FLAGS: {}
+  stick_button:
+    NAME: "stick_button"
+    FLAGS:
+      timeout: 300  # requires longer timeout
+      num_train_tasks: 500  # requires more data
+      min_perc_data_for_nsrt: 1  # requires filtering
+ARGS: []
+FLAGS: {}
+START_SEED: 456
+NUM_SEEDS: 10
+...
+
+# option learning
+---
+APPROACHES:
+  option_learning:
+    NAME: "nsrt_learning"
+    FLAGS:
+      option_learner: "direct_bc"
```

Comment:
what's the difference between putting these here vs in the general flags after the envs?

Reply:
there's no difference, but some of these flags at least feel more related to the approach. i'll move others to the general section